stock_quant_partner_info/models/stock_quant.py

```diff
@@ -14,8 +14,9 @@ class StockQuant(models.Model):
 
     @api.depends('location_id', 'history_ids')
     def _compute_partner_id(self):
-        for quant in self.filtered(
-                lambda q: q.location_id.usage in ['customer', 'supplier']):
-            move = quant.history_ids[:-1]
+        for quant in self:
+            move = quant.history_ids.sorted(key=lambda m: m.date)[-1:]
```

Comment:
you can get > 1 move here, and this will crash on line 20 below as reported byt someone else.

Reply:
Not in this case. With this you always get one record. If you make `[:-1]` it's when you get more records.

Reply:
@pedrobaeza What I am fixing is https://github.com/OCA/stock-logistics-warehouse/pull/291#issuecomment-311901848